packages/block-library/src/comments-pagination-next/block.json

```
@@ -38,6 +38,23 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
```

```suggestion
			"style": true
```

To make the border controls optional (hidden by default) we can just omit this default controls config.